### ISSUE TITLE
WebP is widely available now - add default support?

### ISSUE THREAD

**Issue by foucdeg** [SmileyChris/easy-thumbnails#655]
WebP is widely available now. We are considering full adoption without a PNG / JPG fallback, and we'd like easy-thumbnails to only generate a WebP thumbnail and not bother with jpg / png.

The explanation given on https://easy-thumbnails.readthedocs.io/en/latest/ref/webp/ made perfect sense when it was written, but now, I think more and more developers will consider using only webp.

Are there any plans to add a WebP-first mode to easy-thumbnails?

**Comment by benzkji** [comment]
it's already there, you would just use the setting `THUMBNAIL_EXTENSION = "webp"` and get webp everywhere. Or is this not what you would want?

**Comment by foucdeg** [comment]
@benzkji great to hear that it's supported! Indeed the docs aren't clear about webp support. If a maintainer could update them, I'd be happy to update my config based on the new docs.

**Comment by benzkji** [comment]
https://easy-thumbnails.readthedocs.io/en/latest/ref/settings/#easy_thumbnails.conf.Settings.THUMBNAIL_EXTENSION

and even:  
https://easy-thumbnails.readthedocs.io/en/latest/ref/settings/#easy_thumbnails.conf.Settings.THUMBNAIL_TRANSPARENCY_EXTENSION

**Comment by robertpro** [comment]
Yep, simply doing this fixes everything, no need to do anything else.

```python
THUMBNAIL_EXTENSION = "webp"
THUMBNAIL_TRANSPARENCY_EXTENSION = "webp"
```
